Marlin/Configuration.h

```diff
@@ -2415,7 +2415,7 @@
  * - If stepper drivers sleep, XY homing may be required again before Z homing.
  */
 #if BOTH(KNUTWURST_BLTOUCH, KNUTWURST_4MAXP2)
-  #define Z_SAFE_HOMING
+  //d#define Z_SAFE_HOMING
```

Comment:
?

Reply:
Since the 4MaxPro1+2 have on the left side these "clean-brushes", why driving to the center and back to clean the nozzle.